server/models/Itinerary.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+const Cars = require("./Cars")
+const Flights = require("./Flights")
+const Hotels = require("./Hotel")
+const User = require("./User")
+
+const itinerarySchema = new Schema({
+  user:[User],
```

Comment:
so the user should be an actual reference, because we want the itinerary to be linked to the user. The rest are good though!